logstash-core/spec/logstash/pipeline_spec.rb

```diff
@@ -53,6 +72,10 @@ def close
   end
 end
 
+class DummyOutputMore < DummyOutput
```

Comment:
Let me check If I can remove this and still expose the same problem. I wanted to replicate the situation I had locally with my metric branch.